examples/paho-mqtt/Makefile

```diff
@@ -16,7 +16,17 @@ QUIET ?= 1
 
 USEPKG += paho-mqtt
 
+CFLAGS += -DESP_WIFI_SSID=\"define yours\" -DESP_WIFI_PASS=\"secret\"
+
+ifeq ($(BOARD),esp32-wrover-kit)
+USEMODULE += esp_gdb esp_idf_heap esp_log_tagged esp_log_startup esp_log_colored
```

Comment:
Please don't enable personal preferences by default even though they might be helpful. The standard behavior should be the UNIX way. `esp_idf_heap` is enabled automatically if `esp_wifi` is enabled.
```suggestion
```